include/eos/core/LandmarkMapper.hpp

```diff
@@ -102,8 +103,7 @@ class LandmarkMapper
      *
      * @param[in] mappings A set of landmark mappings.
      */
-    LandmarkMapper(const std::unordered_map<std::string, std::string>& mappings) : landmark_mappings(mappings)
-    {};
+    LandmarkMapper(std::unordered_map<std::string, std::string> mappings) : landmark_mappings(std::move(mappings)) {};
```

Comment:
May I ask for the reason for this small change and the one above to the constructor? Is it more efficient?

Reply:
Honestly, I am not sure. Now when I look at this code I think it may be a good idea to add `explicit` modifier (because it is only one argument), but there is no reason to make this `std::move` change.